mm2src/mm2_bitcoin/script/src/sign.rs

```diff
@@ -737,7 +746,7 @@ mod tests {
             shielded_spends: vec![],
             shielded_outputs: vec![],
             zcash: false,
-            posv: true,
+            posv: true, // TODO: Is this intended? In coins file for coin "tBLK", with this prefix 111, the field "pos" is true, not "posv".
```

Comment:
Yes, this is intended as it was part of this PR https://github.com/KomodoPlatform/komodo-defi-framework/pull/1815 for Reddcoin and Potcoin.